File: Expressions/budget.c
#include <stdio.h>

int main(void){
    float income, rent, utilities, groceries, transportation, savings, expenses, spend;
    float prent, putilities, pgroceries, ptransportation, psavings, pexpenses, pspend;
    printf("This is a budget calculator.\n How much do you make a month?\n");
    scanf("%f", &income);
    printf("how much do your rent cost?\n");
    scanf("%f", &rent);
    printf("how much do your utilities cost?\n");
    scanf("%f", &utilities);
    printf("how much do your groceries cost?\n");
    scanf("%f", &groceries);
    printf("how much do your transportation cost?\n");
    scanf("%f", &transportation);
    savings = income * .2;
    expenses = rent + utilities + groceries + transportation;
    spend = income - expenses - savings;
    printf("You make $%.2f\n", income);
    printf("your expenses are $%.2f\n", expenses);
    printf("your savings are $%.2f\n", savings);
    printf("your spending money is $%.2f\n", spend);
    prent = rent/income *100;
    putilities = utilities/income *100;
    pgroceries = groceries/income *100;
    ptransportation = transportation/income *100;
    psavings = savings/income *100;
    pexpenses = expenses/income *100;
    printf("your rent is %d%% of your income \n", (int) prent);
    printf("your utilities is %d%% of your income \n", (int) putilities);
    printf("your groceries is %d%% of your income \n", (int) pgroceries);
    printf("your transportation is %d%% of your income \n", (int) ptransportation);
    printf("your savings is %d%% of your income \n", (int) psavings);
    printf("your expenses is %d%% of your income \n", (int) pexpenses);
    return 0;
}

File: Expressions/budget.py
print("Hello! This is a financail caculator!")
income = float(input("how much do you make a month?:\n"))
rent = float(input("how much doess your rent cost?:\n"))
utilities = float(input("what does your utilities cost?:\n"))
groceries = float(input("how much does your groceries cost?:\n"))
trasnportation = float(input("how much does your trasportation cost:\n"))
print("you make", income)
savings = income*0.2
expenses= rent + utilities + groceries + trasnportation
spending = income-expenses-savings
print("your expenses are", expenses)
print("your rent is", rent)
print("your utilities are", utilities)
print("your groceries are", groceries)
print("your trasnportation cost is", trasnportation)
prent = (float(rent)/float(income))*100
print("your rent is", prent)
putilities = (float(utilities)/float(income))*100
pgroceries = (float(groceries)/float(income))*100
ptrasnportation = (float(trasnportation)/float(income))*100
print("your uitlities is", putilities)
print("your groceries is", pgroceries)
print("your trasnportation is", ptrasnportation)